docs/docs/installation.md

```diff
@@ -98,4 +98,6 @@ set MUSPINSIM_WITH_OPENMP=1
 
     ```bash
     pip install ./muspinsim --user
-    ```
\ No newline at end of file
+    ```
+
+> **For developers:** You may also use `python setup.py --with-openmp build_ext --inplace` to compile the source with OpenMP while developing.
```

Comment:
```suggestion
!!! note "For developers"
    You may also use `python setup.py --with-openmp build_ext --inplace` to compile the source with OpenMP while developing.
```

I think this is good, just wondering if we should use admonition here as well? It's a new tool to me so feel free to ignore or use a different type (chose `note` as it gives a different colour to `important` above)

Reply:
@patrick-austin We certainly could, the only reason I didn't was because I wasn't sure about using them on the theory pages where they would look like
![image](https://user-images.githubusercontent.com/90245114/212896722-ac5fec4c-b5c2-4fa2-be66-09e09b1ce8b3.png)
I thought it made them seem clearer which may not be the desired effect if its only for developers but I am happy to change all of them, what do you think?

Reply:
I see your point about highlighting information that is likely not to be relevant for most users - in which case let's leave it as is. As long as it's a consistent approach I'm happy.